pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -417,51 +412,117 @@ curl -H "Content-Type: application/json" \
 http://localhost:8845
 ```
 
-It will take time for your collator to sync with the local Paseo Relay chain.
-
-We now need to onboard the parachain to Paseo.
-
-Go to the Parachains tab.
-
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.37.32 AM.png" alt=""><figcaption><p>Parachains Tab</p></figcaption></figure>
-
-Select "+ ParaId" and make sure to use the stash account:
-
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.38.09 AM.png" alt=""><figcaption><p>Reserve ParaId</p></figcaption></figure>
-
-Next, select "+ ParaThread". Make sure to use the stash account, upload your wasm and genesis state:
-
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.38.50 AM.png" alt=""><figcaption><p>Register ParaThread</p></figcaption></figure>
-
-It will take a moment for the ParaThread to onboard:
-
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.39.09 AM.png" alt=""><figcaption><p>ParaThread Onboarding</p></figcaption></figure>
+Ensure the chain is synced with Paseo:
+```bash
+2024-12-10 09:06:05 [Relaychain] Warp sync is complete, continuing with state sync.    
+2024-12-10 09:06:06 [Relaychain] State sync is complete, continuing with block sync.    
+2024-12-10 09:06:06 [Relaychain] 🏆 Imported #47 (0xb04f…12a4 → 0x6adf…6582)    
+2024-12-10 09:06:06 [Relaychain] 🏆 Imported #48 (0x6adf…6582 → 0xd036…e502) 
+```
 
-Once onboarded, we now need to use sudo privileges (Alice account) on the Paseo Relay chain to "force" a lease for our parachain so that it can begin to use produce blocks.
+We now need to onboard the chain to Paseo.
 
-Go to the "sudo" tab:
+### Reserve Para ID
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.45.44 AM.png" alt=""><figcaption><p>Sudo tab</p></figcaption></figure>
+We can reserve a para ID for the chain using pop cli:
 
-Use sudo privileges on the Relay chain to force a lease for our parachain:
+```bash
+pop call chain --url ws://localhost:57731
+```
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.46.09 AM.png" alt=""><figcaption><p>Force a lease</p></figcaption></figure>
+ 
+```bash
+┌   Pop CLI : Call a chain
+│
+◇  What would you like to do?
+│  Reserve a parachain ID 
+│
+◇  Signer of the extrinsic:
```

Comment:
We could include `--use-wallet` to exemplify a more solid flow.
Otherwise we can come back once the feature its merged!